static/openapi/prerendering-spec.json

```diff
@@ -1533,274 +1970,207 @@
-                        }
-                      },
-                      "required": ["status", "data"]
+                          "caching": {
+                            "type": "boolean",
+                            "description": "Whether or not to cache rendered pages."
+                          },
+                          "ttl": {
+                            "type": "number",
+                            "description": "Cache lifetime of rendered pages (in seconds)."
+                          },
+                          "created": {
+                            "type": "string",
+                            "format": "date-time",
+                            "description": "Date when the origin was created."
+                          },
+                          "lastUpdated": {
+                            "type": "string",
+                            "format": "date-time",
+                            "description": "Date when the origin was updated."
+                          }
+                        },
+                        "required": [
+                          "_key",
+                          "origin",
+                          "siteMaps",
+                          "caching",
+                          "ttl",
+                          "created"
+                        ]
+                      }
                     }
-                  ]
+                  },
+                  "required": ["status", "data"],
+                  "example": {
+                    "status": "OK",
+                    "data": [
+                      {
+                        "_key": "www_origin_com",
+                        "origin": "https://www.origin.com",
+                        "siteMaps": ["/sitemap.xml"],
+                        "identifyNotfound": "Page Not Found",
+                        "allowOriginHeaderValue": "948ef5f9-f15f-4b61-91b3-305b77db30e9",
+                        "passThroughHeaders": ["accept-language", "user-agent"],
+                        "addResponseHeaders": {
+                          "cache-control": {
+                            "basedOnStatusCode": true,
+                            "value": {
+                              "2xx": "public, max-age=86400",
+                              "3xx": "public, max-age=7200",
+                              "4xx": "no-store",
+                              "5xx": "no-store"
+                            }
+                          }
+                        },
+                        "caching": false,
+                        "ttl": 86400,
+                        "created": "2023-01-01T00:00:00.000Z",
+                        "lastUpdated": "2023-02-02T00:00:00.000Z"
+                      },
+                      {
+                        "_key": "sub_origin_com",
+                        "origin": "https://sub.origin.com",
+                        "siteMaps": ["/sitemap.xml"],
+                        "identifyNotfound": "Page Not Found",
```

Comment:
```suggestion
                        "identifyNotfound": "Page not found.",
```

Reply:
This is the text on the website. They generally use something like `Page Not Found` or `PAGE NOT FOUND`, etc...
This is a title, not a description.

Reply:
Ah, gotcha. Do you object to the changing of the case, though?

Reply:
Either way, matching the website is best.